### ISSUE TITLE
[A11y][Fastpass]Aria-label not allowed on b

### ISSUE THREAD

**Issue by ryuyu** [NuGet/NuGetGallery#9319]
URL: https://www.nuget.org/packages/Newtonsoft.Json

Path | #paket-cli > .alert-warning[role="alert"][aria-live="assertive"] > b[display="none"][aria-label="warning"]
-- | --

<b display="none" aria-label="warning"></b>

Fix the following:
aria-label attribute cannot be used on a b with no valid role attribute.

https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html
https://accessibilityinsights.io/info-examples/web/aria-allowed-attr

Looks like the fix here would be to either drop this label altogether, OR give this element a role that would make this label proper usage.

**Comment by camigthompson** [comment]
Verified the issue no longer is flagged by fastpass.
